pytripgui/controller/main_cont.py

```diff
@@ -473,10 +473,27 @@ def on_new_plan(self, event):
         """
         New plan opened from window->plan->New Plan
         """
-        model = self.model
-        from pytripgui.controller.plan_cont import PlanController
-        PlanController.new_plan(model)
-        self.tree.update_tree()
+        logger.debug("add_new_plan() {}".format(None))
+
+        from pytrip.tripexecuter import Plan
+        from pytripgui.plan_vc import PlanQtView
+        from pytripgui.plan_vc import PlanController
+
+        # selected_plan = self._node_obj
+        plan = Plan()
+        plan.basename = self.model.ctx.basename
+        view = PlanQtView()
+        global_kernels = self.model.kernels
+        default_kernel = self.model.kernels[0]  # TODO select default kernel
```

Comment:
Crashes, if no kernels are setup.

Reply:
Is that exception should be handled by app controler, or by plan controler?

Reply:
I'd say plan controller should do something (i.e. cast another exception), finally user should see a popup (or modal) with a message that the kernel(s) should be added

Reply:
I also think plan controller would be more natural.

Reply:
Ok, You can check if i made it right